src/me/StevenLawson/TotalFreedomMod/Commands/Command_tagnyan.java

```diff
@@ -12,6 +15,11 @@
 @CommandParameters(description = "Gives you a tag with random colors", usage = "/<command> <tag>", aliases = "tn")
 public class Command_tagnyan extends TFM_Command
 {
+    public static final List<String> FORBIDDEN_WORDS = Arrays.asList(new String[]
+    {
+        "admin", "owner", "moderator", "developer", "console"
```

Comment:
Can this not be pulled from another list? I am sure we already have defined a load of these sorts of blocked words??